dspace-api/src/test/java/org/dspace/core/ContextTest.java

```diff
@@ -310,6 +310,9 @@ public void testIsReadOnly() throws SQLException
         Context instance = new Context(Context.Mode.READ_ONLY);
         assertThat("testIsReadOnly 1", instance.isReadOnly(), equalTo(true));
 
+        //When in read-only, we only support abort().
```

Comment:
Do we? In the same commit you removed the check for being in read only mode when doing a database rollback.